extlinks/links/migrations/0013_linkevent_urlpattern_urlpattern_collections.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.2.14 on 2024-08-20 16:33
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("organisations", "0008_alter_collection_id_alter_organisation_id_and_more"),
+        ("links", "0012_alter_linkevent_id_alter_linksearchtotal_id_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="linkevent",
+            name="urlpattern",
```

Comment:
you'll need to add the content_type & object ID fields here instead of url_pattern